helm_deploy/laa-crown-court-proceeding/values-dev.yaml

```diff
@@ -44,23 +44,16 @@ service:
   targetPort: 8087
 
 ingress:
-  external_whitelist_source_range: "128.77.75.64/26, 18.169.147.172/32, 35.176.93.186/32, 18.130.148.126/32, 35.176.148.126/32, 5.172.196.188, 13.232.220.164, 23.22.2.46, 23.83.129.219, 23.92.127.2, 23.106.37.99, 23.111.152.74, 23.111.159.174, 37.252.231.50, 43.225.198.122, 43.229.84.12, 46.20.45.18, 46.246.122.10, 50.2.185.66, 50.16.153.186, 52.0.204.16, 52.24.42.103, 52.48.244.35, 52.52.34.158, 52.52.95.213, 52.52.118.192, 52.57.132.90, 52.59.46.112, 52.59.147.246, 52.62.12.49, 52.63.142.2, 52.63.164.147, 52.63.167.55, 52.67.148.55, 52.73.209.122, 52.89.43.70, 52.194.115.181, 52.197.31.124, 52.197.224.235, 52.198.25.184, 52.201.3.199, 52.209.34.226, 52.209.186.226, 52.210.232.124, 54.68.48.199, 54.70.202.58, 54.94.206.111, 64.237.49.203, 64.237.55.3, 66.165.229.130, 66.165.233.234, 72.46.130.18, 72.46.131.10, 76.72.167.154, 76.72.172.208, 76.164.234.106, 76.164.234.130, 82.103.136.16, 82.103.139.165, 82.103.145.126, 85.195.116.134, 89.163.146.247, 89.163.242.206, 94.75.211.73, 94.75.211.74, 94.247.174.83, 96.47.225.18, 103.10.197.10, 103.47.211.210, 104.129.24.154, 104.129.30.18, 107.182.234.77, 108.181.70.3, 148.72.170.233, 148.72.171.17, 151.106.52.134, 159.122.168.9, 162.208.48.94, 162.218.67.34, 162.253.128.178, 168.1.203.46, 169.51.2.18, 169.54.70.214, 169.56.174.151, 172.241.112.86, 173.248.147.18, 173.254.206.242, 174.34.156.130, 175.45.132.20, 178.162.206.244, 178.255.152.2, 178.255.153.2, 179.50.12.212, 184.75.208.210, 184.75.209.18, 184.75.210.90, 184.75.210.226, 184.75.214.66, 184.75.214.98, 185.39.146.214, 185.39.146.215, 185.70.76.23, 185.93.3.65, 185.136.156.82, 185.152.65.167, 185.180.12.65, 185.246.208.82, 188.172.252.34, 190.120.230.7, 196.240.207.18, 196.244.191.18, 196.245.151.42, 199.87.228.66, 200.58.101.248, 201.33.21.5, 207.244.80.239, 209.58.139.193, 209.58.139.194, 209.95.50.14, 212.78.83.12, 212.78.83.16"
-  internal_whitelist_source_range: "128.77.75.64/26, 18.169.147.172/32, 35.176.93.186/32, 18.130.148.126/32, 35.176.148.126/32, 35.178.209.113, 3.8.51.207, 35.177.252.54, 35.177.15.253, 35.178.3.79, 35.177.131.251"
```

Comment:
I have a feeling we want to keep the `internal_whitelist_source_range`. It's used in the _ingress.yaml_ template and hasn't been updated as part of this PR (unless it's that change which is missing from this PR?).

Reply:
Looking back on previous PRs for our other services, we've removed the whitelist. However, I can't see why we would want to do that. Ideally we just want Cloud Platform, VPN users and Pingdom to have access - I believe this is what's contained in the old `external_whitelist_source_range`, but we will need to confirm this is still correct. If possible we should also add comments to specify why each IP is present.

[Here](https://dsdmoj.atlassian.net/browse/LCAM-1272) is the original whitelist PR. The `128.77.75.64/26` IP seems to be for the Prima VPN, PR [here](https://dsdmoj.atlassian.net/browse/LASB-3555).

Reply:
The allowlist has been added back in for non production environments.